### DISCUSSION TITLE
Idea add "Notify me when network connection is lost"

### DISCUSSION THREAD

**Discussion by markpahulje** [GyulyVGC/sniffnet#622] [Ideas]
Firstly, wanted to say I love this app and great simple design. May I suggest one minor improvement to "Notify me when network connection is lost". Moreover, perhaps a dropped log would be nice as well. I work remote and trying to advise my network provided of time(s) when my connection is dropping and how long it's up for.

**Comment by GyulyVGC** [comment]
Hi @markpahulje, thanks for the suggestion. 

I'm moving this to an issue so that I can easily track this feature request!

**Reply by GyulyVGC** [reply]
#632

**Reply by markpahulje** [reply]
Thank you.
